Dockerfile

```diff
@@ -1,49 +1,45 @@
-FROM quay.io/aptible/ubuntu:14.04
+FROM alpine:3.6
 
 # Install NGiNX.
-RUN apt-get update && \
-    apt-get install -y software-properties-common \
-    python-software-properties && \
-    add-apt-repository -y ppa:nginx/stable && apt-get update && \
-    apt-get -y install curl ucspi-tcp apache2-utils ruby
+RUN apk update && \
```

Comment:
I would look at some of our CI Dockerfiles and try to combine all your `RUN` statements together to help minimize image size.

Reply:
Yeah, I combine as much as reasonably possible for that setup step.  Trying to stay in sweet spot of minimizing the layers while making it easy to pull something from cache if needed.